src/utils/_variables.scss

```diff
@@ -0,0 +1,18 @@
+$gap-size: 1px;
+$border-color: #000;
+$color-square: #eee;
+$calendar-margin: auto;
+$font-size: 30px;
+$border-size: 1px;
```

Comment:
Don't create variables for variables. You use it only in one place, so you don't need a separate variable for this.
Check all your variables and delete redundant ones